LMGeocoder/LMGeocoder.m

```diff
@@ -14,6 +14,11 @@
 #define kGoogleAPIReverseGeocodingURL(lat, lng) [NSString stringWithFormat:@"https://maps.googleapis.com/maps/api/geocode/json?latlng=%f,%f&sensor=true", lat, lng];
 #define kGoogleAPIGeocodingURL(address)         [NSString stringWithFormat:@"https://maps.googleapis.com/maps/api/geocode/json?address=%@&sensor=true", address];
 #define kGoogleAPIURLWithKey(url, key)          [NSString stringWithFormat:@"%@&key=%@", url, key];
+// final format for country is e.g components=country:DE
+#define countryCodeParam @"components=country"
+#define languageParam @"language="
+#define kGoogleAPIURLAppendCountry(url, params) [NSString stringWithFormat:@"%@&%@:%@", url, countryCodeParam, params];
+#define kGoogleAPIURLAppendLanguage(url, params)[NSString stringWithFormat:@"%@&%@=%@", url, languageParam, params];
```

Comment:
general one: I'd recommend using methods (or in case of defining static values constants.) instead of defines. They are very hard to debug and its not very readable (especially for global defines).